x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -243,7 +272,7 @@ function restructureConnectorsThatSupportIncident(
             subAction: 'pushToService',
             subActionParams: {
               incident: {
```

Comment:
Is there potential data loss on `externalId`?

Reply:
`externalId` is only use from Case. it is not possible to have it there